icons/calendar-cog.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "karsa-mistmere",
```

Comment:
Can you add the people from the original calendar and cog icon you used here?

Reply:
I have already added karsa-mistmere and ericfennis that are the contributors to the two icons I based mine on.

Reply:
Ohh, looks like I messed up and didn't add myself when I redid the cogs in #1225